skin.json

```diff
@@ -6,9 +6,24 @@
   "localization": {
     "defaultLanguage": "en",
     "availableLanguageFile": [
-      {"language": "en", "languageFile": "//player.ooyala.com/static/v4/stable/4.1.4/skin-plugin/en.json"},
-      {"language": "es", "languageFile": "//player.ooyala.com/static/v4/stable/4.1.4/skin-plugin/es.json"},
-      {"language": "zh", "languageFile": "//player.ooyala.com/static/v4/stable/4.1.4/skin-plugin/zh.json"}
+      {
+        "language": "en",
+        "languageFile": "//player.ooyala.com/static/v4/stable/4.1.4/skin-plugin/en.json",
+        "androidResource": "skin-config/en.json",
+        "iosResource": "skin-config/en.json"
```

Comment:
dmitry, are you sure this is the resource path for iOS?  I think it is probably just "en.json" or something like that